src/cljc/athens/common_events/resolver/atomic.cljc

```diff
@@ -379,6 +379,36 @@
     tx-data))
 
 
+(defmethod resolve-atomic-op-to-tx :shortcut/new
+  [db {:op/keys [args]}]
+  (let [{page-name :name}    args
+        page-uid             (common-db/get-page-uid db page-name)
+        reindex-shortcut-txs (->> (common-db/get-sidebar-elements db)
```

Comment:
Why is it necessary to reindex? These should not be out of order, and we're only adding at the end anyway.

Reply:
yeah you are right no need for reindex  here